### ISSUE TITLE
Layout/EmptyLinesAroundArguments false positive

### ISSUE THREAD

**Issue by dvandersluis** [bbatsov/rubocop#5511]
I'm trying to DRY up some code where I have a case statement where each condition appends to a variable using `<<`. When extracting the append outside of the case statement, I then get a false positive from `Layout/EmptyLinesAroundArguments`.

Three situations that _do not_ cause the offense:
1. Move the `out <<` to each branch of the case statement
2. Remove the blank lines in the case statement
3. Remove `out <<` altogether or replace it with `out =`

--------

## Expected behavior

No offense should be reported

## Actual behavior

An offense is reported for every blank line in the case statement.

## Steps to reproduce the problem

```ruby
out << case var
  when :foo
    'yes'

  when :bar, :baz
    'no'
end
```

## RuboCop version

```
$ rubocop -V
0.52.1 (using Parser 2.4.0.2, running on ruby 2.3.3 x86_64-darwin16)
```

**Comment by pocke** [comment]
It has been fixed in the master branch. Probably related: https://github.com/bbatsov/rubocop/pull/5340
Thank you!
